README.md

```diff
@@ -78,52 +78,20 @@ You can also use `Ctrl` + `Alt` + `Shift` + `M` shortcut to open it.
 ![](docs/open_git_machete.gif)
 
 
-### Discover
+### Branch graph
 
-The `machete` file describes relations between branches in your repository.
-These relations are probably determined by the order of branch creation &mdash;
-which branch has been created from which &mdash; but this is not a strict rule).<br/>
-It'll be located under `.git/machete` path in your repository.
+For each branch, Git Machete indicates the relation to each of its child branches.
+If the edge between them is **green** that means the child branch is in sync with its parent branch &mdash; in other words, there are no commits in the parent branch that don't belong to the child.
+But if there are some commits in the parent branch that are **not** reachable from the child, then the edge is **red** &mdash; you need to [rebase](#rebase) child branch onto the parent.
+Edge can also be **gray** or **yellow** &mdash; that means respectively the branch was merged to the parent, or a fork point can't be determined automatically
```

Comment:
Maybe just split this sentence into two - one for gray, one for yellow (occurrence of `mdash`/`respectively`/`that == the latter` in a single sentence is a good indicator for split)